### ISSUE TITLE
extract ivector model from kaldi

### ISSUE THREAD

**Issue by yml-bit** [idiap/kaldi-ivector#6]
Hello, author.Have you considered separating Ivector from Kaldi?When I was trying to separate the ivector from Kaldi, I found that the calculation of statistics and the extraction of ivector depended on base/util/ GMM/HMM, etc. I got confused during the debugging process.Do you have any good methods or suggestions for this?

**Comment by mrsrikanth** [comment]
Do you mean that you want to compute I-vectors without using Kaldi?

**Comment by yml-bit** [comment]
Yes, I think so.Kaldi is highly integrated, making it difficult to read and understand the Ivector code.Do you have any suggestions for the independence of ivector？



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Srikanth M ***@***.***&gt;; 
发送时间: 2021年3月11日(星期四) 下午4:42
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [idiap/kaldi-ivector] extract ivector model from kaldi (#6)





 
Do you mean that you want to compute I-vectors without using Kaldi?

**Comment by mrsrikanth** [comment]
I'm not aware of something like that (mostly because I didn't look for it). But this is something that might soon come [here](https://www.github.com/idiap/pkwrap).

**Comment by yml-bit** [comment]
Ok,thanks a lot.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Srikanth M ***@***.***&gt;; 
发送时间: 2021年3月11日(星期四) 下午5:42
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [idiap/kaldi-ivector] extract ivector model from kaldi (#6)





 
I'm not aware of something like that (mostly because I didn't look for it). But this is something that might soon come here.
